client/my-sites/plugins/marketplace/marketplace-test/index.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import React, { useEffect, useState } from 'react';
```

Comment:
Reviewing this file can be ignored :)

Reply:
Just to confirm, we're planning to delete it once this is finalized, right?

Reply:
It will be deleted... However, I cannot promise you when we will delete it. But I can promise you that the customer will never see it. 😄

Reply:
These are famous last words 😉 

But really, I'm concerned that someone developing another feature could borrow from this code and use it in their work, ultimately leading to this code running in production, which may be undesirable.

Reply:
Ok I understand. let's get it reviewed/fixed! I will also put in an explicit notice in a few places 👍